### ISSUE TITLE
[Feat]: Add Album Tag to Music Video NFO

### ISSUE THREAD

**Issue by Hendo16** [RandomNinjaAtk/docker-lidarr-extended#206]
The generated Music Video NFO doesn't include an album tag that relates to what album the video is promoting.

If we can already match a video title to a song track then we should also be able to get the Album information, and include that tag within the NFO.

If the script is only pulling singles, maybe looking at the all releases for that song and grabbing the first release that's listed as an Album might be a workaround.

**Comment by RandomNinjaAtk** [comment]
This is not likely to happen, the only way to make it work is by essentially using guesswork and I prefer not to do that...

**Comment by Hendo16** [comment]
Fair enough - spent the day working on a python script that's gone through all of my videos and has accurately matched the title with the recording to fetch the album. I'll chuck it up on a repo soon, admittedly there is a small level of guesswork but out of 200+ songs if it identifies an album its correct. It'd be good to use something similar so we can get music videos based on what's in the library. Wouldn't be perfect but having the option there would mean that those who choose to name their files correctly could have that functionality

**Comment by RandomNinjaAtk** [comment]
If you want to write a PR and add it as an option to the existing script, I would merge it in.

**Comment by Hendo16** [comment]
I'll take a look at how i'd integrate it into the current video script and see what i can do

**Comment by RandomNinjaAtk** [comment]
Just make sure to add an option/parameter so it can be enabled/disabled.
